angular.js

```diff
@@ -4,5 +4,59 @@ module.exports = {
         'plugin:@angular-eslint/recommended--extra',
         'plugin:@angular-eslint/template/process-inline-templates',
         './typescript'
-    ]
+    ],
+    rules: {
+        /*
+            Overrides to Angular recommended rules:
+            https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.json
+        */
+
+        '@angular-eslint/component-class-suffix': 'error',
+        '@angular-eslint/contextual-lifecycle': 'error',
+        '@angular-eslint/directive-class-suffix': 'error',
+        '@angular-eslint/no-conflicting-lifecycle': 'error',
+        '@angular-eslint/no-empty-lifecycle-method': 'error',
+        '@angular-eslint/no-host-metadata-property': 'error',
+        '@angular-eslint/no-input-rename': 'error',
+        '@angular-eslint/no-inputs-metadata-property': 'error',
+        '@angular-eslint/no-output-native': 'error',
+        '@angular-eslint/no-output-on-prefix': 'error',
+        '@angular-eslint/no-output-rename': 'error',
+        '@angular-eslint/no-outputs-metadata-property': 'error',
+        '@angular-eslint/use-lifecycle-interface': 'warn',
+        '@angular-eslint/use-pipe-transform-interface': 'error',
+
+        /*
+            Overrides to Angular extra recommended rules:
+            https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
+        */
+
+        'no-restricted-imports': [
+            'error',
+            {
+                paths: [{
+                    name: 'rxjs/Rx',
+                    message: 'Please import directly from \'rxjs\' instead'
+                }]
+            }],
+        '@typescript-eslint/member-ordering': [
+            'error',
+            {
+                default: [
+                    'static-field',
+                    'instance-field',
+                    'static-method',
+                    'instance-method'
+                ]
+            }],
+        'no-restricted-syntax': [
+            'error',
+            {
+                selector: 'CallExpression[callee.object.name=\'console\'][callee.property.name=/^(debug|info|time|timeEnd|trace)$/]',
+                message: 'Unexpected property on console object was called'
+            }],
+        '@typescript-eslint/no-inferrable-types': ['error', { ignoreParameters: true }],
+        '@typescript-eslint/no-non-null-assertion': 'error',
+        'no-fallthrough': 'error'
```

Comment:
[no-fallthrough](https://eslint.org/docs/rules/no-fallthrough)

**Proposal**
This rule can be ignored, because we have already [evaluated it](https://github.com/airbnb/javascript/blob/d5d406a849f03b8ce1bc8848acb7ef1690d29c85/packages/eslint-config-airbnb-base/rules/best-practices.js#L132).